assets/css/_ui_kit.scss

```diff
@@ -278,6 +279,27 @@ $color-tooltip-background: #333333;
   }
 }
 
+@mixin button-primary-new {
+  color: $white;
+  padding: 8px;
+  background-color: new_tokens.$eggplant-500;
+  border-radius: 4px;
+
+  &:hover {
+    background-color: #5d3092;
```

Comment:
I'm pretty sure this hex code is a value that bootstrap came up with based on `$eggplant-500`. I did briefly have this dream of just getting rid of this and just using the bootstrap logic, but that messed up the c-drawer-tab on the route ladders page.

I think there's a Real Solution™ hiding out somewhere, but I'm hoping this is an acceptable incremental change 😅